### PR TITLE
information_schema should not fail for illegal storage format

### PR DESCRIPTION
**Pull request by VicoWu**
Hi @electrum  @kokosing 
I have just migrated the original PR(https://github.com/prestodb/presto/pull/11847) from prestodb to prestosql.

### CODE REVIEW

**Comment by VicoWu** [comment]
@electrum  @kokosing 
Do you have any ideas about where is this kind of table(without storageformat info) created? I try to create table in presto like `create table abc( a int)` , but it does have storageformat infomation;

**Comment by VicoWu** [comment]
@kokosing @electrum 
I have updated the test case for it.The test logic is:
I try to create a hive table with storage format whose serde, inputformat and outputformat is null . After created , I try to retrieve this table, it should be retrieved succesfully;
Is this logic right?

**Comment by electrum** [comment]
It is not possible to create this through Presto. You would need to use the metastore API directly or possibly use Hive (could be done with a product test).

**Comment by kokosing** [comment]
See io.prestosql.tests.utils.QueryExecutors#onHive, with this you can pass SQL to hive.

File: presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java [hunk 1]
```diff
@@ -108,6 +108,7 @@
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.hive.metastore.TableType;
+import org.iq80.leveldb.table.TableBuilder;
```
Comment (electrum) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java]: This was accidentally imported and is causing the build to fail. Please remove it.

**Comment by electrum** [comment]
@VicoWu Do you want to fix the build failure? Otherwise I can fix it up and merge it

**Comment by VicoWu** [comment]
@electrum  The build error has been fixed. Thanks

**Comment by electrum** [comment]
Thanks! Can you squash the commits?

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**